lua/entities/gmod_tardis_interior/modules/sh_screens.lua

```diff
@@ -39,12 +39,13 @@ ENT:AddHook("Initialize", "screens", function(self)
 	if screens then
 		self.screens3D={}
 		for k,v in pairs(screens) do
-			self.screens3D[k]=TARDIS:LoadScreen(k, {
-				width=v.width,
-				height=v.height,
-				ext=self.exterior,
-				int=self,
-				visgui_rows=v.visgui_rows
+			self.screens3D[k] = TARDIS:LoadScreen(k, {
+				width = v.width,
+				height = v.height,
+				ext = self.exterior,
+				int = self,
+				visgui_rows = v.visgui_rows,
+				power_off_black = v.power_off_black,
```

Comment:
This should default to true (unless you disagree)?

Reply:
It is true by default

Reply:
Are you sure? It doesn't look like it only when explicitly set by the screen

Reply:
You're right, sorry, I was thinking about another thing. Fixed now.